src/helpers/eventBridge.ts

```diff
@@ -122,12 +128,12 @@ export default class EventBridge {
       })
       .promise();
 
-    await this.getEvents(); // need to clear this manual published event from the SQS observer queue.
+    await this.getEvent(); // need to clear this manual published event from the SQS observer queue.
 
     return result;
   }
 
-  async getEvents(): Promise<SQS.ReceiveMessageResult | undefined> {
+  async getEvent(): Promise<SQS.ReceiveMessageResult | undefined> {
```

Comment:
The name could be made even more explicit, e.g. `getLatestEvent`, `getMostRecentEvent` or something similar. Thoughts?

Reply:
Yep totally agree, sorry for the delay 🥸